jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildTarTask.java

```diff
@@ -151,6 +152,10 @@ public void buildTar()
       throw new GradleException(
           "container.appRoot is not an absolute Unix-style path: " + ex.getInvalidPathValue(), ex);
```

Comment:
These exceptions for purely for syntactically invalid config values. `/dev/pts/0` is OK, since it is syntactically correct Unix path. No semantics involved.